src/images/dynamodb_local.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{core::WaitFor, Image};
 
 const NAME: &str = "amazon/dynamodb-local";
-const TAG: &str = "latest";
```

Comment:
If I remember correctly, this was purposefully set to `latest` because `latest` always refers to what is deployed on AWS so testing your app against that makes sense.

Reply:
Ah I see. cool then. will drop that commit.